### PR TITLE
Df 2074 6

### PR DESCRIPTION
**Pull request by shaunmillargov**
# Description

This PR includes the following proposed change(s):

- {DF-2074}

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Ran all unit tests plus enhanced GET DF Payload unit test. 

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [] I have made corresponding changes to the documentation
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: digitalforms-api/src/main/java/ca/bc/gov/open/digitalformsapi/viirp/controller/DfPayloadsApiDelegateImpl.java [hunk 1]
```diff
@@ -7,51 +7,59 @@
...
+			 src = dfPayloadService.getDFPayload(noticeNo, correlationId);
+			
+		} catch (ApiException ex) {
+			logger.error("Failure to call DF ORDS, GET DF Payload for notice No: " + noticeNo + " corrleationId: " + correlationId + ". Message: " + ex.getMessage() + " ORDS Response Status Cd: " + ex.getCode());
+			throw new DigitalFormsException(ex.getMessage());
+		}
+			
+		try { 
+			// Transfer from ORDS Client Library GetDFPayloadServiceResponse to DF GetDFPayloadServiceResponse type. 
+			BeanUtils.copyProperties(src, resp);
```
Comment (karahanb) [inline digitalforms-api/src/main/java/ca/bc/gov/open/digitalformsapi/viirp/controller/DfPayloadsApiDelegateImpl.java:57]: Should we do a null check and response status check of src before copying the properties to the response?

**Review by karahanb — APPROVED** [review]
Great work!

File: digitalforms-api/src/main/java/ca/bc/gov/open/digitalformsapi/viirp/service/VipsRestServiceImpl.java [hunk 2]
```diff
@@ -45,7 +45,7 @@ public GetCodetablesServiceResponse getCodeTableValues(String correlationId) {
                 .headers(headers -> headers.setBasicAuth(properties.getVipsRestApiUsername(), properties.getVipsRestApiPassword()))
                 .retrieve()
                 .bodyToMono(GetCodetablesServiceResponse.class)
-                .doOnSuccess(System.out::println)
+                //.doOnSuccess(System.out::println)
```
Comment (karahanb) [inline digitalforms-api/src/main/java/ca/bc/gov/open/digitalformsapi/viirp/service/VipsRestServiceImpl.java:48]: I think we should change this to log both success and failure cases at information level with the current logger that we use at some point.

Comment (shaunmillargov) [reply digitalforms-api/src/main/java/ca/bc/gov/open/digitalformsapi/viirp/service/VipsRestServiceImpl.java:48]: Agreed. I commented out this line as it was clogging up the Splunk output. Every time the code tables is requested, we get lines and lines of output in Splunk.

File: digitalforms-api/src/main/java/ca/bc/gov/open/digitalformsapi/viirp/controller/DfPayloadsApiDelegateImpl.java (hunk 1 above)
Comment (shaunmillargov) [reply digitalforms-api/src/main/java/ca/bc/gov/open/digitalformsapi/viirp/controller/DfPayloadsApiDelegateImpl.java:57]: Good point. I'll do a test and see how BeanUtils deals with one or the other object being null.

**Pull request merged**